_posts/minutes/2016-03-18-minutes.markdown

```diff
@@ -1 +1,35 @@
+---
+---
+Begin: 01:05pm 
+### Introduction
+##A-Reports: 
```

Comment:
Since this will be converted to a heading, I don't think we need the colon at the end.